app/code/Magento/Cron/Console/Command/CronCommand.php

```diff
@@ -135,10 +136,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $params[ProcessCronQueueObserver::STANDALONE_PROCESS_STARTED] = $bootstrapOptionValue;
             }
         }
+
         /** @var Cron $cronObserver */
         $cronObserver = $objectManager->create(Cron::class, ['parameters' => $params]);
         $cronObserver->launch();
-        $output->writeln('<info>' . 'Ran jobs by schedule.' . '</info>');
+
+        // phpcs:ignore Magento2.Functions.DiscouragedFunction.Discouraged
+        if (posix_isatty(STDOUT)) {
```

Comment:
Hello @fredden,

just checking the documentation of [stream_isatty](https://www.php.net/manual/en/function.stream-isatty.php),  can we replace `posix_isatty` with it.

Reply:
I've made this change. `stream_isatty()` is also a "discouraged" function.